fastlane/Fastfile

```diff
@@ -8,16 +8,277 @@ platform :android do
-    get_prs_list(repository:GHHELPER_REPO, start_tag:"#{options[:start_tag]}", report_path:"#{File.expand_path('~')}/simplenote-android_prs_list.txt")
+    get_prs_list(repository:GHHELPER_REPO, start_tag:"#{options[:start_tag]}", report_path:"#{File.expand_path('~')}/simplenoteandroid_prs_list.txt")
+  end
+
+  #####################################################################################
+  # download_metadata_string 
+  # -----------------------------------------------------------------------------------
+  # This lane downloads the translated metadata (release notes,
+  # app store strings, title, etc.) from GlotPress and updates the local files
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # fastlane download_metadata_string build_number:<build_number> version:<version>
+  #
+  # Example:
+  # fastlane download_metadata_string build_number:573 version:10.3
+  #####################################################################################
+  desc "Downloads translated metadata from GlotPress"
+  lane :download_metadata_strings do |options| 
+    values = options[:version].split('.')
+    files = {
+      "release_note_#{values[0].to_s.rjust(2, "0")}#{values[1]}" => {desc: "changelogs/#{options[:build_number]}.txt", max_size: 0},
+      play_store_promo: {desc:"short_description.txt", max_size: 80},
+      play_store_desc: {desc:"full_description.txt", max_size: 0},
+      play_store_app_title: {desc:"title.txt", max_size: 50}
+    }
+
+    delete_old_changelogs(build: options[:build_number])
+    download_path=Dir.pwd + "/metadata/android"
+    gp_downloadmetadata(project_url: "https://translate.wordpress.com/projects/simplenote/android/release-notes/", 
+      target_files: files, 
+      locales: SUPPORTED_LOCALES.map {| hsh | [ hsh[:glotpress], hsh[:google_play] ]},
+      source_locale: "en-US",
+      download_path: download_path)
+    
+    android_create_xml_release_notes(download_path: download_path, build_number: "#{options[:build_number]}", locales: SUPPORTED_LOCALES.map {| hsh | [ hsh[:glotpress], hsh[:google_play] ]})
+    add_us_release_notes(relese_notes_path: download_path + "/release_notes.xml", version_name: options[:version])
+    sh("git add #{download_path} && (git diff-index --quiet HEAD || git commit -m \"Update metadata translations for #{options[:version]}\") && git push")
+  end 
+
+  #####################################################################################
+  # build_bundle
+  # -----------------------------------------------------------------------------------
+  # This lane builds an app bundle
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane build_bundle [skip_confirm:<skip confirm>]
+  #####################################################################################
+  desc "Builds an app bundle"
+  lane :build_apk do | options |
```

Comment:
Is there a reason we are building an apk rather than an app bundle? I guess we just haven't gotten around to that yet.

Reply:
Yep, we are still releasing the full APK.  There isn't a large size savings when using app bundle for Simplenote Android.  The app is only 3.8MB at the moment and we would reduce the download size by about 12%.

Reply:
Makes sense. We may want to make the change at some point anyway for consistency but not a big deal.

Reply:
Oh yes, just something to do.